### PR TITLE
Update stackoverflow question count

### PR DESCRIPTION
**Pull request by FabianLauer**
There's [~27k questions](https://stackoverflow.com/search?q=docker+is%3Aquestion) by now, not just 7k.  :wink:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ However, there might be a way to implement that feature *on top of* Docker.
   <tr>
     <td>Stack Overflow</td>
     <td>
-      Stack Overflow has over 17000 Docker questions listed. We regularly
+      Stack Overflow has over 27000 Docker questions listed. We regularly
```
Comment (cpuguy83) [inline CONTRIBUTING.md]: How about just `thousands`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ However, there might be a way to implement that feature *on top of* Docker.
   <tr>
     <td>Stack Overflow</td>
     <td>
-      Stack Overflow has over 17000 Docker questions listed. We regularly
+      Stack Overflow has over 27 thousand Docker questions listed. We regularly
```
Comment (cpuguy83) [inline CONTRIBUTING.md]: I meant changing form such a specific number like 27k, to simply "Stack Overflow has thousands of Docker questions listed."

Comment (FabianLauer) [inline CONTRIBUTING.md]: Oh, sure

**Review by AkihiroSuda — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -137,7 +137,7 @@ However, there might be a way to implement that feature *on top of* Docker.
   <tr>
     <td>Stack Overflow</td>
     <td>
-      Stack Overflow has over 17000 Docker questions listed. We regularly
+      Stack Overflow has over thousands of Docker questions listed. We regularly
```
Comment (thaJeztah) [inline CONTRIBUTING.md]: "over thousands" needs to be "thousands" 😊

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (FabianLauer) [inline CONTRIBUTING.md]: Oh gawd, I'm so sorry 🙈 
I owe you one guys.

**Comment by thaJeztah** [comment]
Thanks @FabianLauer. Can you squash your commits, so that there's a single commit in this PR? You can find some instructions here; https://docs.docker.com/opensource/workflow/work-issue/#pull-and-rebase-frequently, but let us know if you need help 👍

**Comment by tonistiigi** [comment]
ping @FabianLauer

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**